src/components/NoteDetails/NoteDetailsSnackbar.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react';
+import Button from '@material-ui/core/Button';
+import Snackbar from '@material-ui/core/Snackbar';
+
+export default function NoteDetailsSnackbar({
+  error,
+  saveNoteName,
+  saveNoteContent,
+}) {
+  useEffect(() => {
+    if (error) {
+      setOpen(true);
+    }
+  }, [error]);
+
+  const [open, setOpen] = React.useState(false);
+
+  const handleClose = (event, reason) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+
+    setOpen(false);
+
+    if (error.text.includes('name')) {
+      saveNoteName();
+    }
+    if (error.text.includes('content')) {
```

Comment:
Having logic based around `error.text` includes `'something'` doesn't look like a good idea.
For example a better approach would be to have
```
errors = {
  saveNoteName: ...,
  saveNoteContent: ...,
  createNote: ...,
  };
  ```
  Then here we would just do
  ```
  if (errors.saveNoteContent) {
     ...
  }
  ```
Using JS variables is more robust and clear, rather than relying on the error text string having some word included into it.